build.sh

```diff
@@ -103,7 +103,12 @@ sed -i \
 	"${HOME}/go/src/github.com/mattermost/mattermost-server/Makefile" \
 	"${HOME}/go/src/github.com/mattermost/mattermost-server/build/release.mk"
 make --directory="${HOME}/go/src/github.com/mattermost/mattermost-server" \
-	config-reset build-linux package-linux \
+	config-reset \
+	BUILD_NUMBER="dev-$(go env GOOS)-$(go env GOARCH)-${MATTERMOST_RELEASE}" \
+	GO="GOARCH= GOOS= $(command -v go)" \
+	PLUGIN_PACKAGES=''
```

Comment:
I'm not sure this actually needs to be here at all. the `package` step in Mattermost's build script is already running the same command that this target runs (see the build-release.patch modification above). Unless both the root copy and the dist copy need to be reset? I don't know.